Sources/Client.swift

```diff
@@ -91,17 +92,16 @@ open class Client {
         }
     }
 
-    fileprivate func URLForFragment(fragment: String = "", parameters: [String: Any]? = nil) -> URL? {
-        if var components = URLComponents(string: "\(scheme)://\(server)/spaces/\(spaceIdentifier)/\(fragment)") {
+    fileprivate func URL(forComponent component: String = "", parameters: [String: Any]? = nil) -> URL? {
```

Comment:
correct. fileprivate is the same as private, except if two classes are in the same file for instance, the second class would have access.